views/fe-utils/graphTableRowDataUtil.js

```diff
@@ -0,0 +1,15 @@
+import { TOTAL_LABEL } from '../../constants'
+import { studyCountsToPercentage } from './helpers'
+
+export const graphTableRowData = (site) => {
+  const rowData = []
+  site.forEach((site) => {
```

Comment:
there are 2 variables named `site` so we will want to change one of them.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/193/commits/32c8504a19d74b69df56ae08b713b68c8eda7771